### PR TITLE
Evening League Enhancements

### PR DESCRIPTION
**Pull request by hndrewaall**
This PR is a grab-bag of tweaks to make things shiny for EL. Included:

- Make 'this episode' language configurable
- Add win ratios stats to dashboard
- Minor cosmetic and refactory things

Screenshot:
image

### CODE REVIEW

File: app/league/dashboard/models.py [hunk 1]
```diff
@@ -239,20 +245,24 @@ def episode_stats(cls, episode=None, season=None, num_players=5):
...
             else:
                 wins[game.black.id] = wins.get(game.black.id, 0) + 1
+
             games_played[game.white.id] = games_played.get(game.white.id, 0) + 1
             games_played[game.black.id] = games_played.get(game.black.id, 0) + 1
-            stones_given[game.white.id] = stones_given.get(game.white.id, 0) \
-                + (game.handicap)
+
+            stones_given[game.white.id] = \
+                stones_given[game.white.id] + (game.handicap)
```
Comment (wisygig) [inline app/league/dashboard/models.py]: +1

File: app/league/dashboard/models.py [hunk 2]
```diff
@@ -262,6 +272,8 @@ def episode_stats(cls, episode=None, season=None, num_players=5):
                 kyus_killed[game.black.id] = \
                     kyus_killed.get(game.black.id, 0) + 1
 
+        win_ratios = {p.id: wins[p.id] / games_played[p.id] for p in players}
```
Comment (wisygig) [inline app/league/dashboard/models.py]: divide by zero?

Comment (hndrewaall) [inline app/league/dashboard/models.py]: hahahahaa uhhhhhhhhhh you make an excellent point

Comment (hndrewaall) [inline app/league/dashboard/models.py]: though, it doesn't seem to mind: 
image

Comment (hndrewaall) [inline app/league/dashboard/models.py]: I suppose as long as the numerator is 0, it just calls it 0?

Comment (hndrewaall) [inline app/league/dashboard/models.py]: image

hmmmmmmmmmmmmmmm

Comment (hndrewaall) [inline app/league/dashboard/models.py]: image
oh wait no there we go 😬

Comment (wisygig) [inline app/league/dashboard/models.py]: that's terrifying.

```{p.id: wins[p.id] / games_played[p.id] for p in players if games_played[p.id] > 0}```?

Comment (hndrewaall) [inline app/league/dashboard/models.py]: yup

**Pull request merged**